src/test/java/team/serenity/logic/commands/studentinfo/AddScoreCommandTest.java

```diff
@@ -0,0 +1,161 @@
+package team.serenity.logic.commands.studentinfo;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static team.serenity.commons.core.Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX;
+import static team.serenity.commons.core.Messages.MESSAGE_NOT_VIEWING_A_GROUP;
+import static team.serenity.commons.core.Messages.MESSAGE_NOT_VIEWING_A_LESSON;
+import static team.serenity.commons.core.Messages.MESSAGE_SCORE_NOT_WITHIN_RANGE;
+import static team.serenity.commons.core.Messages.MESSAGE_STUDENT_NOT_FOUND;
+import static team.serenity.testutil.Assert.assertThrows;
+import static team.serenity.testutil.TypicalIndexes.INDEX_FIRST;
+import static team.serenity.testutil.TypicalIndexes.INDEX_SECOND;
+import static team.serenity.testutil.TypicalStudent.AARON;
+import static team.serenity.testutil.TypicalStudent.JOHN;
+
+import org.junit.jupiter.api.Test;
+
+import team.serenity.commons.core.index.Index;
+import team.serenity.logic.commands.CommandResult;
+import team.serenity.logic.commands.exceptions.CommandException;
+import team.serenity.model.group.student.Student;
+
+class AddScoreCommandTest {
+    @Test
+    public void constructor_nullParameter_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new AddScoreCommand((Index) null, 1));
+        assertThrows(NullPointerException.class, () -> new AddScoreCommand((Student) null, 1));
+    }
+
+    @Test
+    public void execute_addScoreOutOfRange_failure() throws CommandException {
```

Comment:
Instead of execute_addScoreOutOfRange_failure(), maybe it should be named as execute_addScoreOutOfRange_throwsCommandException()

Reply:
Ah yes, thanks for noticing, I will make the changes:)